Source/common/SNTBlockMessage.m

```diff
@@ -109,7 +109,12 @@ + (NSString *)stringFromHTML:(NSString *)html {
   return [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
 }
 
-+ (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event {
++ (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event customURL:(NSString *)url {
```

Comment:
nit: probably worth commenting on the behavior for url with "null" vs. other strings that result in an invalid URL.

Reply:
Done. In the process I also extended it to allow format strings in the custom URL and documented that goo.